source/GameInterface/Services/LoadingScreen/Interfaces/ICoopLoadingScreen.cs

```diff
@@ -0,0 +1,381 @@
+﻿using System;
+using TaleWorlds.Core;
+using TaleWorlds.Engine;
+using TaleWorlds.Engine.GauntletUI;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.ScreenSystem;
+using TaleWorlds.TwoDimension;
+
+namespace GameInterface.Services.LoadingScreen.Interfaces;
+
+/// <summary>
+/// Interface for CoopLoadingScreen
+/// </summary>
+public interface ICoopLoadingScreen
```

Comment:
Since this isn't unit testable, you can remove this interface and directly instantiate `CoopLoadingScreen` in `CoopLoadingScreenHandler`